### DISCUSSION TITLE
SIP48 Discussion

### DISCUSSION THREAD

**Discussion by ohager** [signum-network/CIPs#74] [Ideas and Proposals]
> This is the first time we experiment with the discussion feature...

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community to share thoughts and ideas of new platform features.

We are starting off with [SIP48](#73 ) where we propose "Alias Expiry". We recommend to read the SIP carefully and you can start asking questions, making suggestions etc.

## Brief Summary

Atm, Aliases are assigned forever unless they are traded. This allows Alias Farmers to occupy a lot of Aliases and slightly misuse the idea. Think of Aliases as identifiable and user owned containers with mutable on-chain data. Together with [SRC44](https://github.com/signum-network/SIPs/blob/master/SIP/sip-44.md) and [SRC47](https://github.com/signum-network/SIPs/blob/master/SIP/sip-47.md) very nice things can be done here. The SIP48 suggests a way to have an expiry with a renewable service similar to common TLD services.

**Comment by pir8radio** [comment]
looks like the sip "discussion" is for all sips, not per sip.  i just created a sip 46 poll and it put it under the discussion tab, but i see you named discussions as if they were only for sip 48..     so may want to make the sip discussion title/info more generic to cover all sips?
